test/changes/url.test.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @author https://github.com/acvetkov
+ */
+
+import {changesDetailUrl, changesListUrl} from '../../src/changes/url';
+
+describe('changes/url', function () {
+
+    describe('changesListUrl', function () {
+
+        it('should return empty locator', function () {
+            assert.equal(changesListUrl(), 'changes/?build=');
+        });
+
+        it('should return list apiPath for build id', function () {
+            assert.equal(changesListUrl('my-build-id'), 'changes/?build=id:my-build-id');
+        });
+    });
+
+    describe('changesDetailUrl', function () {
+
+        it('should return apiPath without locator', function () {
+            assert.equal(changesDetailUrl(), 'changes/');
+        });
+
+        it('should return detail apiPath for id', function () {
+            assert.equal(changesDetailUrl('1'), 'changes/id:1');
+        });
+
```

Comment:
I mean can you add tests to Changes class to cover withDetails case?

Builds/Artifacts classes has not tests, because they has not additional logic, except url creation. But Changes class has it in withDetails logic branch. It will be great to cover withDetails case, feel free for choose http mock lib, for example nock.